drivers/video/fbdev/mxsfb.c

```diff
@@ -1466,6 +1466,11 @@ static int mxsfb_init_fbinfo(struct mxsfb_info *host)
 
 	fb_info->fix.line_length =
 		fb_info->var.xres * (fb_info->var.bits_per_pixel >> 3);
+#if !defined(CONFIG_SOC_IMXRT)
```

Comment:
It seems the conflict was resolved incorrectly during cherry-picking. This change must be moved down, to line 1478, refer to the original commit 7283d34.

Reply:
After fixing, remove the commits 73ee10b and 7ca78ba

Reply:
Done